WebDriverAgentLib/Utilities/FBScreen.m

```diff
@@ -19,17 +19,13 @@ + (double)scale
   return [XCUIScreen.mainScreen scale];
 }
 
-+ (CGSize)statusBarSizeForApplication:(XCUIApplication *)application
++ (CGSize)statusBarSize
 {
-  XCUIApplication *app = XCUIApplication.fb_systemApplication;
-  // Since iOS 13 the status bar is no longer part of the application, it’s part of the SpringBoard
-  XCUIElement *mainStatusBar = app.statusBars.allElementsBoundByIndex.firstObject;
+  XCUIElement *mainStatusBar = XCUIApplication.fb_systemApplication.statusBars.allElementsBoundByIndex.firstObject;
   if (nil == mainStatusBar) {
     return CGSizeZero;
   }
-  CGSize result = mainStatusBar.frame.size;
-  // Workaround for https://github.com/appium/appium/issues/15961
-  return CGSizeMake(MAX(result.width, result.height), MIN(result.width, result.height));
```

Comment:
it looks like even without these lines, ios 17 returned proper value. Checking with an ios 16

Reply:
ios 15.8 also worked